### PR TITLE
fix: Extend row hash to 128 bits to guarantee unique row id

### PR DESCRIPTION
**Pull request by TrevorBergeron**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Comment by tswast** [comment]
> ERROR tests/system/small/ml/test_llm.py::test_create_embedding_generator_multilingual_model

Looks like a test flake. Will raise an issue with our on-call this week.

File: bigframes/session/__init__.py [hunk 1]
```diff
@@ -832,9 +834,13 @@ def _read_gbq_table(
                     ordering=ordering,
                 )
             else:
-                array_value = self._create_total_ordering(table_expression)
+                array_value = self._create_total_ordering(
+                    table_expression, table_size=table.num_rows
```
Comment (tswast) [inline bigframes/session/__init__.py:838]: Could you please try this with a VIEW type table? I don't think `num_rows` is always populated.

File: bigframes/session/__init__.py [hunk 2]
```diff
@@ -1455,10 +1461,18 @@ def _create_empty_temp_table(
     def _create_total_ordering(
         self,
         table: ibis_types.Table,
+        table_size: int,
```
Comment (tswast) [inline bigframes/session/__init__.py:1464]: Likewise, we might need to handle missing size here.

**Review by tswast — CHANGES_REQUESTED** [review]

File: bigframes/session/__init__.py (hunk 2 above)
Comment (TrevorBergeron) [reply bigframes/session/__init__.py]: added.

File: bigframes/session/__init__.py (hunk 1 above)
Comment (TrevorBergeron) [reply bigframes/session/__init__.py]: yes, this won't be populated for views or potentially with streaming inserts. defaulting to assume table is large and need 128 bits.

**Review by tswast — APPROVED** [review]

**Pull request merged**